webpack.config.dist.js

```diff
@@ -7,8 +7,8 @@ module.exports = (env, options) => {
     config.cache = false;
     config.output.path = __dirname + '/dist';
     config.entry = {
-        'webgl-operate': ['require.ts', 'polyfill.ts', 'webgl-operate.ts'],
-        'webgl-operate.slim': ['require.ts', 'polyfill.ts', 'webgl-operate.slim.ts'],
+        'webgl-operate': ['require.ts', 'webgl-operate.ts'],
+        'webgl-operate.slim': ['require.ts', 'webgl-operate.slim.ts'],
```

Comment:
I'd suggest to add polyfill entry points for dist and lib:
`'webgl-operate-polyfills': ['polyfill.ts'],`